README.md

```diff
@@ -96,20 +96,74 @@ Initialize the `BUYClient` with your credentials from the *Mobile App Channel*
 BUYClient *client = [[BUYClient alloc] initWithShopDomain:@"yourshop.myshopify.com"
                                                    apiKey:@"aaaaaaaaaaaaaaaaaa"
                                                 channelId:@"99999"];
+```
 
-// Fetch your products
-[self.client getProductsPage:1 completion:^(NSArray *products, NSUInteger page, BOOL reachedEnd, NSError *error) {
-        if (error) {
-                NSLog(@"Error retrieving products: %@", error.userInfo);
-        } else {
-                for (BUYProduct *product in products) {
-                        NSLog(@"%@", product.title);
-                }
-        }
+### Storefront API
+After initializing the client with valid shop credentials, you can begin fetching collections.
+```objc
+[client getCollections:^(NSArray<BUYCollection *> *collections, NSError *error) {
+	if (collections && !error) {
+		// Do something with collections
+	} else {
+		NSLog(@"Error fetching collections: %@", error.userInfo);
+	}
+}];
+```
+Having a collection, we can then retrieve an array of products within that collection:
+```objc
+BUYCollection *collection = collections.firstObject;
+
+[client getProductsPage:0 inCollection:collection.collectionId completion:^(NSArray<BUYProduct *> *products, NSUInteger page, BOOL reachedEnd, NSError *error) {
+	if (products && !error) {
+		// Do something with products
+	} else {
+		NSLog(@"Error fetching products in collection %@: %@", collection.collectionId, error.userInfo);
+	}
 }];
 ```
-
 Consult the [Usage Section](https://docs.shopify.com/mobile-buy-sdk/ios/integration-guide/#using-the-mobile-buy-sdk) of the Integration Guide on how to create a cart, and checkout with the SDK.
+### Customer API
```

Comment:
Oh, wait, I suppose "Storefront" is meant to include both catalog and checkout. That's probably OK, although it might be nice to separate them.